### PR TITLE
board: initial implementation of `VS201` board, `CC2538 + CC2592`

### PR DESCRIPTION
**Pull request by luisan00**
### Contribution description
Basic implementation of Vision's `VS203` USB, dongle (cc2538+cc2592).

### Testing procedure
Compile and flash the firmware, some example or a test.

### Issues/PRs references
None

### CODE REVIEW

File: boards/vs203/Kconfig [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Copyright (c) 2020 HAW Hamburg
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+
+config BOARD
+    default "meshme" if BOARD_MESHME
```
Comment (eduazocar) [inline boards/vs203/Kconfig:8]: Why the default name of board is not VS203?

**Review by eduazocar — COMMENTED** [review]
I see all good, I have a question before to merge.

**Review by eduazocar — CHANGES_REQUESTED** [review]
Good job :+1: , do squash to merge it

File: boards/vs203/Kconfig (hunk 1 above)
Comment (luisan00) [reply boards/vs203/Kconfig]: copy pasta confusions xD

Comment (luisan00) [reply boards/vs203/Kconfig]: fixed!

**Review by RocioRojas — APPROVED** [review]

**Comment by luisan00** [comment]
ready to go @eduazocar

**Pull request merged**